lib/test/karma.js

```diff
@@ -192,7 +191,10 @@ function karmaRun() {
         })
       })
       .on('close', function() {
-        done();
+        process.nextTick(done);
+      })
```

Comment:
@bguiz After playing around with TeamCity this was the only thing that worked.

I'm happy with this as is but we could also pipe `stderr` if you thought it prudent.